src/main/java/teamfour/tasc/logic/commands/CollapseCommand.java

```diff
@@ -0,0 +1,36 @@
+package teamfour.tasc.logic.commands;
+
+import teamfour.tasc.commons.core.EventsCenter;
+import teamfour.tasc.commons.events.ui.CollapseChangeEvent;
+
+
+public class CollapseCommand extends Command {
+
+    public static final String COMMAND_WORD = "collapse";
+    public static final String MESSAGE_SUCCESS = "Task view collapsed";
+    public static final String MESSAGE_FAILURE = "Already in collapsed view, type \"expand\" to go into expanded view";
```

Comment:
MESSAGE_FAILURE could be MESSAGE_FAILURE_IS_ALREADY_COLLAPSED or something meaningful